content/posts/web-components.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Portable Spectrograms and Verification Interface"
+date: 2025-03-27T22:00:00+10:00
+image: "/images/ecosounds-banner-image.jpg"
+imageCredit: "QUT Ecoacoustics Staff"
+tags:
+    - News
+    - Technical
+---
+
+Open Ecoacoustics have recently published portable web components that can be
+embedded into any website.
+The web components provide Ecoacoustic tools such as spectrograms, annotation
+viewers, and customizable verification grids.
+<!--more-->
+
```

Comment:
let's also overview each of the components, with a working demo:

- spectrogram, with a few variants
- spectrogram with all the chrome, examples of theme change/chrome options changing
- spectrogram with a couple of annotations
- verification grid, including purpose (verify a tag applied to an annotation is correct or not)